.circleci/config.yml

```diff
@@ -71,96 +44,34 @@ jobs:
             JEST_JUNIT_OUTPUT_DIR: ./test-results
       - store_test_results:
           path: ./test-results
-  build_and_push_docker:
-    machine:
-      docker_layer_caching: true
-    steps:
-      - checkout
-      - attach_workspace:
-          at: .
-      - gcp-gcr/build-image:
-          image: $CIRCLE_PROJECT_REPONAME
-          tag: $CIRCLE_SHA1
-          registry-url: us.gcr.io
-      - gcp-gcr/gcr-auth
-      - gcp-gcr/push-image:
-          image: $CIRCLE_PROJECT_REPONAME
-          tag: $CIRCLE_SHA1
-          registry-url: us.gcr.io
-  pulumi_preview:
-    docker:
-      - image: cimg/node:22.16
-    steps:
-      - checkout
-      - pulumi/login
-      - run:
-          name: Install dependencies
-          command: pnpm install --frozen-lockfile
-          working_directory: .infra
-      - run:
-          name: Pulumi preview
-          command: pulumi preview --suppress-outputs --stack dailydotdev/prod -c tag=$CIRCLE_SHA1
-          working_directory: .infra
-  pulumi_up:
-    circleci_ip_ranges: true
-    docker:
-      - image: cimg/node:22.16
-    environment:
-      USE_GKE_GCLOUD_AUTH_PLUGIN: 'True'
-    steps:
-      - checkout
-      - gcp-cli/setup:
-          components: gke-gcloud-auth-plugin
-      - pulumi/login
-      - run:
-          name: Install dependencies
-          command: pnpm install --frozen-lockfile
-          working_directory: .infra
-      - run:
-          name: Pulumi up
-          command: pulumi up -r -y --suppress-outputs --stack dailydotdev/prod -c tag=$CIRCLE_SHA1
-          working_directory: .infra
 workflows:
   build:
     jobs:
-      - build
+      - build-app
       - test:
           requires:
-            - build
+            - build-app
           filters:
             branches:
-              ignore: master
```

Comment:
Noticed that at some point we might've not ran tests on main branch, but because it was named `master` before, this filter was never applied. Not sure if we want to always run it like we're doing now, or don't run on main branch?
@capJavert

Reply:
👍 I'll remove that filter then